libs/api-client-tiger/api/api-client-tiger.api.md

```diff
@@ -7,10 +7,10 @@
 import { AxiosInstance } from 'axios';
 import { AxiosPromise } from 'axios';
 import { IBucket } from '@gooddata/sdk-model';
-import { IDashboardDateFilterConfig } from '@gooddata/sdk-backend-spi';
-import { IDashboardLayout } from '@gooddata/sdk-backend-spi';
+import { IDashboardDateFilterConfig } from '@gooddata/sdk-model';
```

Comment:
Maybe we can now cut the dependency between api-client-tiger and sdk-backend-spi?